_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -12,9 +12,9 @@ accessibility_requirements:
     passed: satisfied
     inapplicable: satisfied
   wcag20:1.3.1: # Info and Relationships (A)
-    secondary: true
+    secondary: This success criterion is **less strict** than this rule. This is because the rule does not ignore irrelevant ARIA properties. This is why some of the failed examples may not satisfy this success criterion.
```

Comment:
Similar comment to below, but shouldn't **less strict** criterion mean some of the failed examples may satisfy the criterion?